### ISSUE TITLE
[Feature Request] HTTP State API

### ISSUE THREAD

**Issue by Hill-98** [Hastwell/Omukade.Cheyenne#4]
I am running a community server, and I want to provide community users with information such as the number of people online on the server, but currently only Shell commands can obtain this information.

I wrote a simple HTTP State API based on Shell code, but it is very simple and I am not familiar with .NET and ASP, so it would be great if you could provide this function!

**Comment by Hastwell** [comment]
This would probably not be very difficult, as Cheyenne is already running
an HTTP server to facilitate WebSocket traffic from clients.

Would these JSON endpoints provide suitable information for your
dashboard/status display/etc? These would be accessed by the Cheyenne
server port (10850), not normal port 80.
GET /api/v1/players => {"playercount":123, "players",["foo","bar"]}
GET /api/v1/players?countonly=true => {"playercount":123}

On Linux, JQ can be used in shell scripts to parse these results and
extract JSON for further use.

**Comment by Hill-98** [comment]
Yes, these JSON APIs look great!

**Comment by Hastwell** [comment]
This feature has been completed. I did change the endpoints slightly to streamline readability and make the slightly less-expensive call the default unless you explicitly want the player names. (The previous default would have sent all player names unless explicitly requesting not to.)

- `GET /api/v1/players => {"count":123}`
- `GET /api/v1/players?names=true => {"count":123, "players":["Foo","Bar"]}`

Closing as complete.
